### PR TITLE
Port SLS app enhancements → OSS

### PR DESCRIPTION
**Pull request by pgobin**
Signed-off-by: PGobz <p.gobin@gmail.com>

### CODE REVIEW

File: core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts [hunk 1]
```diff
@@ -86,7 +86,10 @@ export class EnvLogsSectionComponent implements OnInit {
 	sliderPercentFrom;
 	sliderMax: number = 7;
 	rangeList: Array<string> = ['Day', 'Week', 'Month', 'Year'];
-	selectedTimeRange: string = this.rangeList[0];
+  selectedTimeRange: string = this.rangeList[0];
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts]: indent issue.

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts [hunk 2]
```diff
@@ -41,16 +41,19 @@ export class EnvOverviewSectionComponent implements OnInit {
...
   editBtn:boolean = true;
   saveBtn:boolean = false;
-  saveButton:boolean=false;
+  slsapp:boolean = false;
+  showCncl:boolean=false;
   editButton:boolean=true;
+  saveButton:boolean=false;
+  isCancel:boolean=false;
+  version: string = ">=1.0.0 <2.0.0";
+  isvalid:boolean=false;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: consistent camelCase for variables: isValid, startNew?

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts [hunk 3]
```diff
@@ -73,23 +76,25 @@ export class EnvOverviewSectionComponent implements OnInit {
...
 	errorInclude:any="";
   json:any={};
-  slsapp:boolean = false;
   desc_temp:any;
-  toastmessage:any;
+  private toastmessage: any = '';
   copyLink:string="Copy Link";
   disableSave:boolean = true;
- 
+  is_function:boolean;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: is_function is probably no longer needed.

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts [hunk 4]
```diff
@@ -144,34 +149,47 @@ popup(state){
...
-    this.saveButton = true;
-    this.editButton = false;
-    this.isCancel = false;
-    this.textChanged = true;
-    this.disableSave = true;
-    this.textarea = false;
+  fetchfunction(){
+    this.isLoading = true;
+    this.http.get('/jazz/services/' + this.service.id, null, this.service.id).subscribe(response => {
+      this.is_function = response.data.metadata.is_function_template;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: is_function is going away as we are not allowing edit. it is just view only. no readonly section. maybe dimple's PR takes care of it..

File: core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts [hunk 5]
```diff
@@ -161,20 +164,18 @@ export class EnvLogsSectionComponent implements OnInit {
 	regSelected:string=this.regList[0];
   	public assetSelected:string;
 	public resourceSelected: any;
-	lambdaResourceNameArr;
+  lambdaResourceNameArr;
```
[the remark this answers is not in the document]
Comment (pgobin) [reply core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts]: Addressed.

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts (hunk 3 above)
Comment (pgobin) [reply core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: Addressed.

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts (hunk 2 above)
Comment (pgobin) [reply core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: Addressed.

**Pull request merged**